### PR TITLE
[tests-only][full-ci]bump latest ocis commit id

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
Bump latest ocis commit id.
Part of: https://github.com/owncloud/QA/issues/849

### CODE REVIEW

File: tests/drone/csp.yaml [hunk 1]
```diff
@@ -3,7 +3,8 @@ directives:
     - '''self'''
   connect-src:
     - '''self'''
-    - 'https://${KEYCLOAK_DOMAIN:-keycloak:8443}/'
+    # In contrary to bash and docker the default is given after the | character
```
Comment (phil-davis) [inline tests/drone/csp.yaml:6]:
```suggestion
    # In contrast to bash and docker the default is given after the | character
```

File: tests/drone/csp.yaml [hunk 2]
```diff
@@ -13,14 +14,16 @@ directives:
   frame-src:
     - '''self'''
     - 'https://embed.diagrams.net/'
-    - 'https://${ONLYOFFICE_DOMAIN:-onlyoffice:443}/'
-    - 'https://${COLLABORA_DOMAIN:-collabora:9980}/'
+    # In contrary to bash and docker the default is given after the | character
```
Comment (phil-davis) [inline tests/drone/csp.yaml:17]:
```suggestion
    # In contrast to bash and docker the default is given after the | character
```

File: tests/drone/csp.yaml [hunk 3]
```diff
@@ -13,14 +14,16 @@ directives:
...
+    - 'https://${ONLYOFFICE_DOMAIN|onlyoffice:443}/'
+    - 'https://${COLLABORA_DOMAIN|collabora:9980}/'
   img-src:
     - '''self'''
     - 'data:'
     - 'blob:'
-    - 'https://${ONLYOFFICE_DOMAIN:-onlyoffice:443}/'
-    - 'https://${COLLABORA_DOMAIN:-collabora:9980}/'
+    # In contrary to bash and docker the default is given after the | character
```
Comment (phil-davis) [inline tests/drone/csp.yaml:24]:
```suggestion
    # In contrast to bash and docker the default is given after the | character
```

File: tests/drone/csp.yaml (hunk 1 above)
Comment (PrajwolAmatya) [reply tests/drone/csp.yaml:6]: I copied this from this PR in ocis https://github.com/owncloud/ocis/pull/9028. Should we keep it same or change it?

**Review by saw-jan — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Pull request merged**